jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/vo/AccountVH.java

```diff
@@ -0,0 +1,42 @@
+package com.tmobile.cso.pacman.inventory.vo;
+
+import java.util.List;
+
+public class AccountVH {
```

Comment:
This seems to be CloudTrail Class- No account as the name suggests.

Reply:
CloudTrail and SNS information we need it at Account Level, hence we have included these in AccountVH.

Reply:
That won't be proper POJO defition.

Reply:
We will accommodate review comments.